developer-docs-site/docs/guides/nfts/aptos-token-overview.md

```diff
@@ -1,130 +1,70 @@
+
+In order to create a new NFT collection, a creator calls an existing deployed program to populate a new collection and set of NFTs.
 
 ### Aptos
-The [accounts](../../concepts/accounts.md) in Aptos are homogeneous in that they can store both smart contracts and data. But
-distinct from Ethereum, the associated data of a smart contract is distributed across the space
-of all accounts. The associated data of a contract in Ethereum is broken into pieces
-stored in its owner's account. For example, an NFT smart contract will store all the NFTs and their
-holders in the mapping and other metadata like supply in a state variable; both belong to this
-contract account.
-
-However, in Aptos, NFTs are stored under each owner's account, respectively. Other
-token metadata will be kept under the creator account but not the account owning the contract code.
-It is similar to Solana in the way that the contract code can be written to achieve maximum code
-reuse instead of deploying similar code again and again just for domain separation. Meanwhile, the
-access control mechanism and generics in Move natively enable this pattern and asset security.
-
-Another interesting trade-off between the account models of Ethereum and Solana/Aptos is data locality.
-For Ethereum, it is hard to know all the tokens owned by a specific account but easy to get all the
-owners of a token type. For Solana and Aptos, the opposite is true if no indexing is built. Ethereum
-code can be easily written to do some operations like airdrop to all the holders relying on on-chain
-data only while Solana/Aptos facilitate listing all the tokens an account owns.
+
+The [accounts](../../concepts/accounts.md) in Aptos store both smart contracts and data. Unlike Ethereum, the associated data of a smart contract is distributed across the space of all accounts in [resources](../../concepts/resources.md) within [accounts](../../concepts/accounts.md) or [objects](../../standards/aptos-object.md). For example, a collection and NFT within that collection are stored under distinct objects under different addresses with the smart contract defining them in another. Of course, a smart contract developer could also store data associated with the NFT and collection under the same address as the smart contract or in other objects.
+
+There are two means to create NFTs on Aptos:
+
+* The [no-code standard](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-22.md) allows creators to call into the contract to create new collections and tokens without deploying a new contract.
+* A creator can also customize their NFTs via extensions to the object model by deploying their own contract that can manage all aspects of their collection.
```

Comment:
As with a few other pieces of this document, there is significant tense differences that cause a little thrashing in the wording.  We will want an example for both in the future.

```suggestion
* The [no-code standard](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-22.md) allows creators to call into the contract to create new collections and tokens without deploying a new contract.
* Custom NFT contracts allow creators to customize their NFTs by extending the object model that can manage all aspects of their collection.  The creator would create and deploy their a new contract.
```